torch_geometric/nn/nlp/txt2kg.py

```diff
@@ -0,0 +1,236 @@
+import os
+import time
+from typing import List, Optional, Tuple
+
+import torch
+import torch.multiprocessing as mp
+
+CLIENT_INITD = False
+
+CLIENT = None
+GLOBAL_NIM_KEY = ""
+SYSTEM_PROMPT = "Please convert the above text into a list of knowledge triples with the form ('entity', 'relation', 'entity'). Seperate each with a new line. Do not output anything else. Try to focus on key triples that form a connected graph.”"
+
+
+class TXT2KG():
+    """Uses NVIDIA NIMs + Prompt engineering to extract KG from text.
+    `nvidia/llama-3.1-nemotron-70b-instruct` is on par or better than GPT4o
+    in benchmarks. We need a high quality model to ensure high quality KG.
+    Otherwise we have garbage in garbage out for the rest of the
+    GNN+LLM RAG pipeline.
+
+    Use local_lm flag for local debugging/dev. You still need to be able to
+    inference a 14B param LLM, 'VAGOsolutions/SauerkrautLM-v2-14b-DPO'.
+    Smaller LLMs did not work at all in testing.
+    Note this 14B model requires a considerable amount of GPU memory.
+    """
```

Comment:
It's not clear to me how to use this class. Can we document an example usage (and/or add a reference to the example)?